scripts/3.TCGA-MLexample_Pathway.py

```diff
@@ -341,22 +357,23 @@ def get_threshold_metrics(y_true, y_pred, tissue='all'):
     y_tissue_pred_train = y_pred_train[y_train.index.isin(sample_sub)]
```

Comment:
Are all of these index selections redundant? Could we use a single boolean indexer?